### PR TITLE
FLS-813: HSRA Eligibility Forms updated

### PR DESCRIPTION
**Pull request by hamzabinkhalid**
https://mhclgdigital.atlassian.net/browse/FLS-813

### Change description
- Updated eligibility forms
- Added new forms with eligibility question and logic for HSRA redirect to round based on eligibility question

- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
- Make sure HSRA VR/RP are live
- Go through the login process
- Click on check eligibility button
- Make sure selecting vacancy register or refurbishment project as answer to question "Why are you applying to this funding?" forwards you to the correct Application tasklist for that round

### CODE REVIEW

**Review by nuwan-samarasinghe — CHANGES_REQUESTED** [review]

File: runner/src/server/plugins/engine/page-controllers/ConfirmPageController.ts [hunk 1]
```diff
@@ -20,8 +20,12 @@ export class ConfirmPageController extends SummaryPageController {
             //@ts-ignore
             const state = await adapterCacheService.getState(request);
             const fund_name = state["metadata"]["fund_name"];
-            const round_name = state["metadata"]["round_name"];
-            return redirectTo(request, h, config.eligibilityResultUrl + "/" + fund_name + "/" + round_name);
+            let round_name = state["metadata"]["round_name"];
+            // if it's HSRA fund, get the value of a question from the eligibility form to determine the round
+            if(fund_name === "hsra"){
+                round_name = state["mwumLN"]["OjGBLs"];
```
[the remark this answers is not in the document]
Comment (sfount) [reply runner/src/server/plugins/engine/page-controllers/ConfirmPageController.ts:26]: 👍 outside of a prototype I think we'll want this to work as predictably as possible as well so I think its OK to apply to any fund as long as its configured for it.

The ticket mentions it working similarly to the `markAsComplete` sections which get added but are treated uniquely by the code.

Something like:
```javascript
const redirectToEligibleRound = state["redirectToEligibleRound"]

const url = new URL(`/${fund_name}/${round_name}`, config.eligibilityResultUrl)


if (redirectToEligibleRound) {
  // include the appropriate round to redirect to if it was included in the funds eligibility questions
  url.searchParams.set('redirect_to_eligible_round', redirectToEligibleRound)
}

return redirectTo(request, h, url.href)
```

**Review by sfount — COMMENTED** [review]
Really impressive proving the concept and getting the configuration started! Outside of the prototype I think it would be good if the redirect feature could be split out into another PR to make the git history a bit easier to follow and so people can review the two changes separately

**Review by nuwan-samarasinghe — DISMISSED** [review]
LGTM

for HSRA config are we not using FAB

**Comment by hamzabinkhalid** [comment]
> LGTM
> 
> for HSRA config are we not using FAB

No, Since we already had HSRA configs in the codebase, trying to update those for the new requirements

**Review by sfount — APPROVED** [review]

**Pull request merged**